main.py

```diff
@@ -4,17 +4,24 @@
 from fastapi.testclient import TestClient
 from fastapi.templating import Jinja2Templates
 from fastapi.staticfiles import StaticFiles
+from datetime import datetime
 
 from database import models, schemas
 from database.database import engine
 from crud import crud
 from database import database
+from pydantic import BaseModel
+
 
 models.Base.metadata.create_all(bind=engine)
 
 app = FastAPI()
 client = TestClient(app)
 
+class TestResult(BaseModel):
```

Comment:
tą klasę sugerowałbym przenieść do database/schemas.py i stamtąd zaimportować

Reply:
I wtedy też usunąć from pydantic import BaseModel

Reply:
Zrobione! Dzięki za sugestię ;) Przeoczyłem to jakoś..